### PR TITLE
Prepare for 0.4.2 release

### PR DESCRIPTION
**Pull request by apyrgio**
Update our changelog to reflect the latest changes in the code, and bump the version from 0.4.1 to 0.4.2.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -5,16 +5,19 @@ All notable changes to this project will be documented in this file.
...
 
 ### Added
 
 - Feature: Add support for HWP/HWPX files (Hancom Office) ([issue #243](https://github.com/freedomofpress/dangerzone/issues/243), thanks to @OctopusET)
 - Platform support: Alpha integration with Qubes OS ([issue #411](https://github.com/freedomofpress/dangerzone/issues/411))
-- Platform support: Debian Trixie (13)
-- Platform support: Ubuntu 23.04 (Lunar Lobster)
+- Platform support: Debian Trixie (13) ([issue #452](https://github.com/freedomofpress/dangerzone/issues/452))
+- Platform support: Ubuntu 23.04 (Lunar Lobster) ([issue #453](https://github.com/freedomofpress/dangerzone/issues/453))
 - Inform about new updates on MacOS/Windows platforms, by periodically checking
```
Comment (deeplow) [inline CHANGELOG.md:18]: Can we put this feature on top? It seems like the most relevant one?

File: CHANGELOG.md [hunk 2]
```diff
@@ -5,16 +5,19 @@ All notable changes to this project will be documented in this file.
...
   our GitHub releases page ([issue #189](https://github.com/freedomofpress/dangerzone/issues/189))
+- Allow users to change their selection from the UI ([issue #428](https://github.com/freedomofpress/dangerzone/issues/428))
```
Comment (deeplow) [inline CHANGELOG.md:20]:
```suggestion
- Allow users to change their document selection from the UI ([issue #428](https://github.com/freedomofpress/dangerzone/issues/428))
```

File: CHANGELOG.md [hunk 3]
```diff
@@ -24,6 +27,10 @@ since 0.4.1, and this project adheres to [Semantic Versioning](https://semver.or
 
 ### Fixed
 
+- Add missing language detection (OCR) models ([issue #357](https://github.com/freedomofpress/dangerzone/issues/357))
+- Replace deprecated `pipes` module with `shlex` ([issue #373](https://github.com/freedomofpress/dangerzone/issues/373), thanks to @OctopusET)
+
```
Comment (deeplow) [inline CHANGELOG.md:32]:
```suggestion
-  Shrink container image with --no-cache option on apk  ([issue #459](https://github.com/freedomofpress/dangerzone/issues/459), thanks to @OctopusET)
```

**Comment by deeplow** [comment]
> - Sanitize potentially unsafe characters from strings that are shown in a user's  terminal.

Can we also link to the PR that fixed it?

**Comment by apyrgio** [comment]
Thanks for the comments :-). I incorporated them in a FIXUP commit.

**Review by deeplow — APPROVED** [review]

**Pull request merged**